config/sunspot.yml

```diff
@@ -3,17 +3,17 @@ production:
     hostname: 127.0.0.1
     port: 8983
     log_level: DEBUG
-    path: /solr/default
+    #path: /solr/default
```

Comment:
Why did this change? This is working in production so I'm worried about changing it